src/AdvancedVI.jl

```diff
@@ -141,6 +141,16 @@ export
 include("objectives/elbo/entropy.jl")
 include("objectives/elbo/repgradelbo.jl")
 
+
+# Variational Families
+export
+    VILocationScale,
```

Comment:
This was a suggestion in the original PR because there exists `Distributions.LocationScale`, which only supports univariate location-scale distributions. Do you think it would be okay to rename it to `LocationScale`?

Reply:
Okay. I'll do `MvLocationScale`.